pkg/audit/reporter.go

```diff
@@ -142,6 +156,13 @@ func (reporter *Reporter) recordAuditSuccessStatus(ctx context.Context, successN
 			failed = append(failed, nodeID)
 			errlist.Add(err)
 		}
+
+		// TODO(kaloyan): Perhaps, this should be executed in the same Tx as overlay.UpdateStats above
+		_, err = reporter.containment.Delete(ctx, nodeID)
```

Comment:
same here. Just because the node passes an audit doesn't mean it should be removed from containment mode. Maybe it still fails to audit the share it was contained for. Again, not sure if my thinking is correct here